src/Groups/directproducts.jl

```diff
@@ -504,13 +530,18 @@ W(g_1,...,g_n, h).
 # Examples
 ```jldoctest
 julia> G = cyclic_group(3)
-Pc group of order 3
+Pc group of order 3 with 1 generator f1
 
 julia> H = symmetric_group(2)
-Sym(2)
+Symmetric group of degree 2 and order 2 with 1 generator
+  (1,2)
 
 julia> W = wreath_product(G,H)
-<group of size 18 with 2 generators>
+Wreath product of
+  pc group of order 3 wr symmetric group of degree 2 and order 2
+with 2 generators
+  WreathProductElement(f1,<identity> of ...,())
+  WreathProductElement(<identity> of ...,<identity> of ...,(1,2))
```

Comment:
I really have no idea what this output should convey. But I think this is one of the types you skipped for now, so please ignore this comment until you tackle this